model-optimizer/extensions/middle/MakeKaldiConstReshapable.py

```diff
@@ -0,0 +1,137 @@
+"""
+ Copyright (C) 2018-2021 Intel Corporation
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+import numpy as np
+
+from mo.middle.replacement import MiddleReplacementPattern
+from mo.front.common.partial_infer.utils import int64_array
+from mo.graph.graph import Graph, Port
+from mo.ops.broadcast import Broadcast
+from mo.ops.concat import Concat
+from mo.ops.const import Const
+from mo.ops.crop import Crop
+from mo.ops.shape import Shape
+
+
+def create_const_with_batch_from_input(input_out_port: Port, second_dim, value=0, precision=np.float):
+    # create const with batch taken from Parameter
```

Comment:
Please, explain all function parameters using the documentation style with `:param` and `:return`.